src/sql/util.py

```diff
@@ -250,56 +250,89 @@ def parse_toml_error(e, file_path):
         return e
 
 
-def get_user_configs(file_path):
+def get_user_configs(primary_path, alternate_path):
     """
     Returns saved configuration settings in a toml file from given file_path
 
     Parameters
     ----------
-    file_path : str
+    primary_path : Path
+        file path to a toml file
+    alternate_path : Path
         file path to a toml file
 
     Returns
     -------
     dict
         saved configuration settings
+    Path
+        the path of the file used to get user configurations
     """
-    data = load_toml(file_path)
-    section_names = ["tool", "jupysql", "SqlMagic"]
-    while section_names:
-        section_to_find, sections_from_user = section_names.pop(0), data.keys()
-        if section_to_find not in sections_from_user:
-            close_match = difflib.get_close_matches(section_to_find, sections_from_user)
-            if not close_match:
-                MESSAGE_PREFIX = (
-                    f"Tip: You may define configurations in "
-                    f"{file_path}. Please review our "
+    data = None
+    tip_displayed = False
+    disable_tip = False
+    for file_path in [primary_path, alternate_path]:
+        if file_path is None or not file_path.exists():
+            continue
+
+        if file_path == alternate_path:
+            STATUS = "Did not find pyproject.toml"
+            if primary_path:
+                STATUS = "Did not find user configurations in pyproject.toml"
+            display.message(
+                f"{STATUS}. " f"Looking for user configurations in {alternate_path}."
```

Comment:
why do we need this line?

Reply:
Currently it looks like this and it looks too verbose:
<img width="1043" alt="Screenshot 2023-11-16 at 8 52 36 PM" src="https://github.com/ploomber/jupysql/assets/29096135/6132c50d-1d2b-4175-9207-c8315642eae9">

I think `Did not find user configurations in pyproject.toml. Loading configurations from /Users/neelashasen/.jupysql/config.` should suffice

Reply:
The other case:

<img width="890" alt="Screenshot 2023-11-16 at 8 54 14 PM" src="https://github.com/ploomber/jupysql/assets/29096135/1b9f00e8-d026-4173-80de-1ecfc13fabc4">

It already mentions `present in /Users/neelashasen/Dev/jupysql_master/jupysql/dummy/pyproject.toml but empty.` but it also displays `Did not find user configurations in pyproject.toml.`. The first line should be sufficient. 

wdyt @edublancas